config/locales/en.yml

```diff
@@ -47,158 +47,126 @@ en:
   funding_widget:
     subjects:
       primary:
-        name: "Primary"
+        name: "Any primary subject"
```

Comment:
I'm not sure whether `primary subject` is the right way to phrase this @emmaachesongray @jenhadfield-dfe - might be worth checking if you haven't already

However, I'd suggest we add a separate ticket on this so not to hold up getting this PR out